core/aws-core/src/main/java/software/amazon/awssdk/awscore/client/builder/AwsDefaultClientBuilder.java

```diff
@@ -187,6 +189,26 @@ private SdkClientConfiguration finalizeAwsConfiguration(SdkClientConfiguration c
                             .build();
     }
 
+    /**
+     * Apply the client override configuration to the provided configuration.
+     */
+    protected SdkClientConfiguration setOverrides(SdkClientConfiguration configuration) {
```

Comment:
Is this method intended to be overridden? If not, can we add `final`? Also, missing override annotation

Reply:
No, it's not, I can add the override and make it final.